### ISSUE TITLE
No HTTPS/TLS for openhardwaremonitor.org

### ISSUE THREAD

**Issue by C0rn3j** [openhardwaremonitor/openhardwaremonitor#964]
Hi, why does the site allow HTTP and the cert name the cert the HTTPS connection presents is wrong?

I can see that the cert name is for *.metanet.ch, and I'm assuming that's where the webhosting is.

For the sysadmin - you can use https://letsencrypt.org/ to get a free certificate and automate the renewal with certbot with a cron job.

**Comment by Thermi** [comment]
I think the website has to support HTTPS, too, because it provides the binaries to users. Without authenticated TLS, it is too easy to inject malicious code with a MITM attack.

**Comment by DarthDestroyer** [comment]
You arent wrong, but that is the smallest problem at this repo ;-)

**Comment by Ajedi32** [comment]
Yeah, as cool as this project is, I'm not comfortable downloading and running unsigned binaries over an unencrypted connection. Couldn't you at least attach the final builds to the GitHub tags so I can download them from GitHub directly?

**Comment by DarthDestroyer** [comment]
Best thing about open source: you can do anything by yourself ;-)
You dont have to download it from the ohm page, you dont have to run unsigned binaries! 
Download the Source, compile and sign it yourself and you are done ;-)

But, you arent really wrong... let us be patient

**Comment by sunk818** [comment]
>  I'm not comfortable downloading and running unsigned binaries over an unencrypted connection

If you're that uncomfortable, maybe you should not use the software. Paranoid much?
